ts/src/decoding/fastPforUnpack.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { describe, expect, it } from "vitest";
+import {
+    fastUnpack32_1, fastUnpack32_2, fastUnpack32_3, fastUnpack32_4,
+    fastUnpack32_5, fastUnpack32_6, fastUnpack32_7, fastUnpack32_8,
+    fastUnpack32_9, fastUnpack32_10, fastUnpack32_11, fastUnpack32_12,
+    fastUnpack32_16,
+} from "./fastPforUnpack";
+
+type Unpacker = (inValues: Int32Array, inPos: number, out: Int32Array, outPos: number) => void;
+
+describe("FastPFOR Unpack Library", () => {
+    const unpackers: Record<number, Unpacker> = {
```

Comment:
There's no point in splitting definition and implementation as this is a constant and doesn't really change or gets manipulated, right?